stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/mention/DefaultUserLookupHandler.kt

```diff
@@ -0,0 +1,86 @@
+package io.getstream.chat.android.ui.message.input.mention
+
+import com.ibm.icu.text.Transliterator
+import io.getstream.chat.android.client.logger.ChatLogger
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.client.models.name
+import java.text.Normalizer
+import kotlin.math.min
+
+private const val MAX_DISTANCE = 3
+private val regexUnaccent = "\\p{InCombiningDiacriticalMarks}+".toRegex()
+
+public class DefaultUserLookupHandler(
```

Comment:
Since it's public and contains some not trivial logic then it's good to add KDOC with description of algorithm